### PR TITLE
Added build script fot jupyter-base-notebook

### PR DESCRIPTION
**Pull request by muskaan2199**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by seth-priya** [comment]
@muskaan2199 please check the travis build failure

File: j/jupyter-base-notebook/jupyter-base-notebook_ubi_8.5.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+./install.sh
+cd ..
+
+git clone  $PACKAGE_URL
+cd notebook
+git checkout $PACKAGE_VERSION
+pip install -e ".[dev,test]"
+jlpm develop
+jupyter server extension enable notebook
+jlpm build
```
Comment (nikhil-kalbande) [inline j/jupyter-base-notebook/jupyter-base-notebook_ubi_8.5.sh:63]: @muskaan2199 - Can we handle failures for build and test separately with exit codes 1 and 2. Please do that

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Comment by nikhil-kalbande** [comment]
@muskaan2199 - 
1. I dont see build_info file in PR. 
2. Also have you validated dockerfile for latest version of jupyter-base-notebook. Do we need changes there?

**Comment by muskaan2199** [comment]
> @muskaan2199 -
> 
> 1. I dont see build_info file in PR.
> 2. Also have you validated dockerfile for latest version of jupyter-base-notebook. Do we need changes there?

@nikhil-kalbande 
I'm waiting for the Dockerfile to be added to build script repository. After that I will add those.
Made changes and added exit conditions.

**Pull request closed without merging**